sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveComparisonTest.scala

```diff
@@ -348,6 +348,7 @@ abstract class HiveComparisonTest
                 queryString.replace("../../data", testDataPath))
               val containsCommands = originalQuery.analyzed.collectFirst {
                 case _: Command => ()
+                case _: InsertIntoTable => ()
```

Comment:
We should not have `InsertIntoTable` inside plan tree when run hive query, looks like this PR breaks something, need some more time to investigate it.